tests/test_cli.py

```diff
@@ -48,3 +50,24 @@ def test_startproject():
         assert result.exit_code == 0
         assert os.listdir() == [project_name]
         assert os.listdir(project_name) == project_files
+
+		
+def test_ColoredFormatter():
+    c = ColoredFormatter()
+    expected = ['DEBUG',
+                '\x1b[34mINFO\x1b[0m',
+                '\x1b[33mWARN\x1b[0m',
+                '\x1b[31mERROR\x1b[0m',
+                '\x1b[30m\x1b[41mCRITICAL\x1b[0m'
+               ]
```

Comment:
pep8 :)
```python
expected = [
    'DEBUG',
    '\x1b[34mINFO\x1b[0m',
    '\x1b[33mWARN\x1b[0m',
    '\x1b[31mERROR\x1b[0m',
    '\x1b[30m\x1b[41mCRITICAL\x1b[0m'
]
```

Reply:
Sorry, a bit new at this.  Should tox be catching these?  I ran it without errors.

Reply:
fixed in latest commit